mono/tests/merp-crash-test.cs

```diff
@@ -376,10 +407,18 @@ public static int Main (string [] args)
 				}
 			}
 
+			// Also test sending a managed exception
+			Exception exception_test_failure = RunManagedExceptionTest ();
+
 			Console.WriteLine ("\n\n##################");
 			Console.WriteLine ("Merp Test Results:");
 			Console.WriteLine ("##################\n\n");
 
+			if (exception_test_failure != null)
+			{
+				Console.WriteLine ("Sending managed exception to MERP failed: {0}\n{1}\n", exception_test_failure.Message, exception_test_failure.StackTrace);
+			}
+
```

Comment:
Couldn't all of this be inside `RunManagedExceptionTest`, too?   And just return 0 or 1 and 
```
    failure_count += RunManagedExceptionTest ();
```

Reply:
Then the awkwardness moves to the for loop and if condition right after this code instead (lines 422-429), at which point one may as well consider changing CrasherClass so it can work for local tests too. I am just trying not to change too much here, but still want test to occur before the "Merp Test Results:" header, and the results to be printed afterwards.

Reply:
Hm I guess the for loop would just not print anything and just have no use, which is fine

Reply:
ah I didn't notice that subtlety.  ok.